### PR TITLE
feat: centralized CI scripts

### PR DESCRIPTION
**Pull request by psturc**
https://issues.redhat.com/browse/HACBS-785

### CODE REVIEW

File: ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-commands.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+git config --global user.email redhat-appstudio-qe-bot@redhat.com
+
+mkdir -p "${HOME}/creds"
+GIT_CREDS_PATH="${HOME}/creds/file"
+git config --global credential.helper "store --file ${GIT_CREDS_PATH}"
+echo "https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com" > "${GIT_CREDS_PATH}"
+
+cd "$(mktemp -d)"
+git clone --no-checkout "https://${GITHUB_TOKEN}@github.com/psturc/e2e-tests.git" .
+git checkout HACBS-784
```
Comment (psturc) [inline ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-commands.sh:26]: once https://github.com/redhat-appstudio/e2e-tests/pull/138 is merged, it needs to be updated to
```suggestion
git clone --no-checkout "https://${GITHUB_TOKEN}@github.com/redhat-appstudio/e2e-tests.git" .
git checkout main
```

**Comment by psturc** [comment]
/test pj-rehearse

Testing whether the PR content is still valid.

@rhopp @flacatus hi, would you have a time to give this PR a look? Thanks.

**Comment by psturc** [comment]
bump

File: ci-operator/step-registry/redhat-appstudio/e2e/OWNERS [hunk 2]
```diff
@@ -0,0 +1,9 @@
+approvers:
+- flacatus
+- rhopp
+- psturc
```
Comment (sawood14012) [inline ci-operator/step-registry/redhat-appstudio/e2e/OWNERS:4]: add sawood14012

**Review by flacatus — APPROVED** [review]

**Pull request merged**